modules/assim.sequential/R/sda.enkf.R

```diff
@@ -802,11 +872,10 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
     
     ## in the future will have to be separated from analysis
     new.state  <- analysis
-    new.params <- params
     
     ANALYSIS[[t]] <- analysis
     
-    if (interactive() & t > 1) { #
+    if (FALSE & interactive() & t > 1) { #
```

Comment:
This FALSE is a bit of a development hack too. Would be nice if there was a way to detect not only if the session is interactive but also whether it has a graphics capability. In lieu of that, probably fine to just drop the ```FALSE &```